README.md

```diff
@@ -154,7 +154,7 @@ Describe your question(s) and [mention](https://docs.github.com/en/github/writin
 [spacex-api]: https://api.spacexdata.com/v3/launches/past
 [spacex-api-docs]: https://docs.spacexdata.com/?version=latest#fce450d6-e064-499a-b88d-34cc22991bcc
 [github-collaborators]: https://help.github.com/en/articles/inviting-collaborators-to-a-personal-repository
-[task-1]: #task-1---javascript
+[task-1]: #task-1---typescript
```

Comment:
is it a correct update?

Reply:
Yes, this updates the link to the task 1, which became `TypeScript` after this PR — https://github.com/adjust/fe-hometask/pull/7.